libecho/src/Echo.cc

```diff
@@ -37,31 +40,91 @@ void Echo::send(const std::vector<uint8_t> &buffer) {
         output->enqueueData(encoded.data() + i, std::min(atOnce, encoded.size() - i));
         auto status = output->getStreamStatus();
         qDebug() << "Status: " << status.first << " " << status.second;
-        output->waitForTick();
+        // TODO: i changed something and this line now freezes the playback after a short time
+        // output->waitForTick();
```

Comment:
It would be nice to fix it before merging. I think it may already be fixed - problem like that was caused by lack of timeout in waitForTick, and no tick was emitted if sound was too short. Increasing output size seemed to fix the problem. Now that I added timeout in waitForTick, everything should be good. Please check